packages/flutter/test/services/raw_keyboard_test.dart

```diff
@@ -568,7 +568,7 @@ void main() {
         'toolkit': 'glfw',
         'keyCode': 340,
         'scanCode': 0x00000032,
-        'codePoint': 0,
+        'codePoint': '',
       });
       final RawKeyEventDataLinux data = shiftLeftKeyEvent.data;
       expect(data.physicalKey, equals(PhysicalKeyboardKey.shiftLeft));
```

Comment:
we still need to add tests for this new code. As far as I can tell the tests don't test the new code.

Reply:
The test that checks for this code is on line 534 https://github.com/flutter/flutter/pull/34752/files#diff-1e6ff69b318c8b31b82b2086c0f8a12eR534. Is this what you are referring to?

Reply:
No, I think he means it's not testing the "multiple code points" part of the code.

Reply:
Thanks, totally missed that. Added test for handling 2 and 3 values

Reply:
Since this is now an int, there's no need for these tests. Existing tests cover the creation of the logical key.